modules/acm/src/main/java/edu/gemini/epics/acm/CaWindowStabilizer.java

```diff
@@ -147,16 +147,20 @@ public void removeListener(CaAttributeListener<T> listener) {
         notifier.removeListener(listener);
     }
 
-    public CaWindowStabilizer<T> reset() {
+    public CaWindowStabilizer<T> restart(Duration settleTime) {
```

Comment:
Maybe we should document this a bit. What does the `settleTime` do?

Reply:
Added documentation. `settleTime` is the time a the value of the filtered `CaAttribute` must be unchanged to be set as the output of the filter.